graphspace/settings/local.py

```diff
@@ -39,9 +39,9 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'test_database',
```

Comment:
Remove this change from the pull request.

Reply:
You can also add this file to .gitignore to avoid accidental inclusions.